app/__init__.py

```diff
@@ -246,8 +247,9 @@ def update_sent_state(sender=None, headers=None, **kwargs):
 @app.errorhandler(500)
 def internal_server_error(error):
     if current_app.config['PROPOGATE_ERROR'] is True:
-        exc = JsonApiException({'pointer': ''}, str(error))
+        raise error
```

Comment:
You have to display appropriate exception at API level and console level too.
In API as Json response and at console using `logger.exception`.

Reply:
Nope, JSON will not be returned in case PROPAGATE_ERROR is True